### ISSUE TITLE
Pokerus spreading

### ISSUE THREAD

**Issue by MatPietrzyk** [Farigh/pokeclicker-automation#153]
### Describe the bug
Pokerus spreading seems to have stopped working after the patch that added sorting of pokemon to breed (could be wrong about the timing, but most likely).

### Steps to Reproduce
With option "Focus on spreading the Pokerus" activated, there is no change in breeding priorities compared to it being off. New pokemon without pokerus aren't being added to the hatchery to be infected, instead the normal priority of breeding efficency is continued.

### Expected behavior
Pokemon without pokerus should be added to hatchery queue to be infected.

**Comment by unproductivityfriendly** [comment]
All my Galar Pokémons got infected from hatching

**Comment by MatPietrzyk** [comment]
Then I do not understand - I have 2 instances of the game running, one on desktop app and one in the Edge browser, and none of them are spreading pokerus correctly. I'm gonna try on some other browsers to see if I can replicate that further.

**Comment by unproductivityfriendly** [comment]
I am running on Chrome 106

**Comment by MatPietrzyk** [comment]
Okay, I have managed to reproduce this error on 2 browsers (chrome and Edge) and on desktop app. It only appears in my playthrough with all of the challenges turned on, in other save with challenges turned off spreading pokerus works perfectly. 
Manually turning off auto attack on a save without the challenges also pretty much stops the spread, with hatchery a lot of time not having any pokemon getting infected and only ocasionally getting pokemon there to spread pokerus to them (and it might be just random from the standard hatchery priority list) - and when I turn on auto attack, it starts working correctly immidiately and there's always a pokemon in there which is getting pokerus. 
In conclusion it might be the problem that is connected to turning off auto attack.

**Comment by MatPietrzyk** [comment]
That might be completely off the mark, but I am just talking about my save which I was running for about 2 hours without any pokemon getting the pokerus on "Experience" focus and "Focus on spreading pokerus" ON.

**Comment by Farigh** [comment]
Could you upload a save reproducing the problem so I can take a look a it ?

**Comment by MatPietrzyk** [comment]
[TEST BROWSER HC.txt](https://github.com/Farigh/pokeclicker-automation/files/9845345/TEST.BROWSER.HC.txt)
Ofc, here it is.

**Comment by Farigh** [comment]
There was indeed a bug, which does not appear to be present in the test setup, which is not ideal ....

It should be fixed now
